modules/references/server/controllers/reference.server.controller.js

```diff
@@ -546,3 +546,32 @@ exports.readMine = async function readMine(req, res) {
   }
   return res.status(200).json(reference);
 };
+
+exports.getCount = async function getCount(req, res, next) {
+  console.log('getCount — userTo:', req.query.userTo); //eslint-disable-line
+  const { userTo } = req.query;
+
+  if (!mongoose.Types.ObjectId.isValid(userTo)) {
+    return res
+      .status(400)
+      .send({ message: 'Missing or invalid `userTo` request param' });
+  }
+
+  try {
+    const query = {
+      userTo: new mongoose.Types.ObjectId(userTo),
+    };
+
+    // Allow non-public references only when userTo is self
+    if (!req.user._id.equals(userTo)) {
+      query.public = true;
```

Comment:
Just a random idea: always show the number of published references. If you look at your profile and there are non-published ones, put an asterisk: (4*)

Reply:
Thanks for the idea!

I gave it a think and concluded:
- It's best to show the same number in the tab than how many experiences they also see in the list. If there's a mismatch, it can get folks confused.
- Asterix would be potentially unclear and we'd need to explain it
- Little more common pattern is a red dot for "unread" or for "notifications" and I think that's a good improvement to call attention to the fact that there are pending experiences.

I added the dot in latest iteration, and update PR description with fresh screenshots. Lemme know what you think!